### PR TITLE
Jfk material fixes

### PR DESCRIPTION
**Pull request by joknu1**
Hi @samtstern,

I've looked into some issues I had with the ss-material-components branch. This is my first PR here, so feel free to give harsh feedback. E.g. Travis did not pass, since I don't have a json for google-services.

Cheers,
Jonas

### CODE REVIEW

File: auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java [hunk 1]
```diff
@@ -33,16 +32,13 @@
...
     private final int mButtonText;
-    private final ForegroundColorSpan mLinkSpan;
 
     private SpannableStringBuilder mBuilder;
 
     private PreambleHandler(Context context, FlowParameters parameters, @StringRes int buttonText) {
         mContext = context;
         mFlowParameters = parameters;
         mButtonText = buttonText;
-        mLinkSpan = new ForegroundColorSpan(ContextCompat.getColor(mContext,
```
Comment (samtstern) [inline auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java:44]: What happened to this span?

**Comment by samtstern** [comment]
@joknu1 thank you for this contribution! Happy to have someone new onboard.

File: auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java (hunk 1 above)
Comment (joknu1) [reply auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java:44]: To my understanding - this span was to get the correct color on the URL links. This seem to be handled by `URLSpan` on it's own - given that `<item name="android:textColorLink">@color/fui_linkColor</item>` is set in the theme.

Comment (samtstern) [reply auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java:44]: 👍 yep!

**Pull request merged**